torch/_inductor/mkldnn.py

```diff
@@ -630,6 +659,8 @@ def fuse_unary(gm: torch.fx.GraphModule):
                 computation_node_input_size = (
                     node.args[0].args[0].meta.get("tensor_meta").shape
                 )
+                if dynamo_config.dynamic_shapes:
+                    computation_node_input_size = None
```

Comment:
for dynamic shape, do you still need the size hint to better decide how to do the prepacking?

Reply:
I think we don't need it, removed.